src/pages/[platform]/build-a-backend/auth/add-social-provider/index.mdx

```diff
@@ -312,9 +312,6 @@ import { signInWithRedirect } from 'aws-amplify/auth';
     await signInWithRedirect({
       provider: 'Apple'
     });
-    const currentUser = await getCurrentUser();
-    const userAttributes = await fetchUserAttributes();
-    console.log({currentUser, userAttributes});
```

Comment:
Missed this from here: https://github.com/aws-amplify/docs/blob/941a617b13c4755fac9608b70a1b4d4866d07ed8/src/pages/gen2/build-a-backend/auth/add-social-provider/index.mdx#L294-L299